docker/travis_test/entrypoint.sh

```diff
@@ -31,7 +31,6 @@ function run_as_normal_user {
         create_db
         export MOJO_LOG_LEVEL=debug
         export MOJO_TMPDIR=$(mktemp -d)
-        export OPENQA_LOGFILE=/tmp/openqa-debug.log
     else
```

Comment:
If we are changing the entrypoint we need to rebuild the docker image

Reply:
yes, but as we're just removing things we set later there is no hurry